llpc/tool/amdllpc.cpp

```diff
@@ -402,6 +402,18 @@ static Result init(int argc, char *argv[], ICompiler **ppCompiler) {
       break;
     }
 
+    // Change defaults of NGG options according to GFX IP
+    if (ParsedGfxIp.major >= 10) {
+      if (ParsedGfxIp.minor >= 3) {
```

Comment:
nit: can you fold these into a single if?

Reply:
Will merge them.